src/components/NavBar/components/NavEventsButtons/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component, PropTypes } from 'react';
```

Comment:
This component was deleted in my prev PR. Why is it up again?

Reply:
maybe rebase make it appear again